### PR TITLE
fix: make SQL editor preview table cells copyable

### LINKED ISSUE #9299: SQL editor preview table cells should be copyable
# Feature request
The cells of the preview table in the SQL editor should have a copy to clipboard button. 

CleanShot 2022-10-03 at 11 04 28@2x

### PR DESCRIPTION
**Pull request by gitstart**
## What kind of change does this PR introduce?

Fixes https://github.com/supabase/supabase/issues/9299

## Feature request
The cells of the preview table in the SQL editor should have a copy to the clipboard button.

## What is the current behavior?
The cells of the preview table in the SQL do not have a copy to the clipboard button.
Screenshot 2022-10-11 at 2 01 46 PM

## What is the new behavior?
Displays copy icon on table cell hover
Screenshot 2022-10-11 at 2 02 45 PM

## Demo

https://user-images.githubusercontent.com/26940294/195341404-a6685b22-6ecc-4695-ad08-5e6d895c9c63.mov

### CODE REVIEW

**Comment by gitstart** [comment]
@saltcod Could you please review this PR for new UI changes?

File: studio/styles/grid.scss [hunk 1]
```diff
@@ -527,6 +527,22 @@
...
+}
+
+.sb-grid-select-cell__formatter:hover .sb-grid-select-cell__copy-action {
+  @apply block opacity-50;
+}
+
+.sb-grid-select-cell__copy-action:hover {
+  @apply opacity-100;
+}
+
```
Comment (Isaiah-Hamilton) [inline studio/styles/grid.scss:545]: Why create its own css class, when you can just put this into the className for that div.

Comment (gitstart) [reply studio/styles/grid.scss]: Thanks @Isaiah-Hamilton we updated PR to use pure tailwindcss classes instead of creating new global css class.

**Comment by saltcod** [comment]
Hey @gitstart works great! 

I'd like to propose a small UI change — I'd like an indication that you've pressed the copy button. Similar to the api settings:  page when you click copy the text changes to "Copied" for a second or two. There isn't room for that text here so maybe change the icon colour to green, or replace with a checkmark for 1-2s? 

CleanShot 2022-10-14 at 10 51 43@2x

CleanShot 2022-10-14 at 10 48 20@2x

**Comment by saltcod** [comment]
Hey @gitstart !  I see activity on this PR — let me know when it's ready for a look and I'll be happy to hop over to it!

**Comment by gitstart** [comment]
Thanks @saltcod, we updated this PR to match your suggestion with showing checkmark icon only. Could you please take a look?

**Review by saltcod — APPROVED** [review]

**Comment by saltcod** [comment]
Merged!  Thanks @gitstart!

**Comment by gitstart** [comment]
This PR was pushed through [Gitstart](https://www.gitstart.com/), with contributions from @Chiazokam, @raph941, @phunguyenmurcul.

**Pull request merged**